lib/std/crypto/aegis.zig

```diff
@@ -203,13 +207,11 @@ fn Aegis128LGeneric(comptime tag_bits: u9) type {
                 blocks[0] = blocks[0].xorBlocks(AesBlock.fromBytes(dst[0..16]));
                 blocks[4] = blocks[4].xorBlocks(AesBlock.fromBytes(dst[16..32]));
             }
-            const computed_tag = state.mac(tag_bits, ad.len, m.len);
-            var acc: u8 = 0;
-            for (computed_tag, 0..) |_, j| {
-                acc |= (computed_tag[j] ^ tag[j]);
-            }
-            if (acc != 0) {
-                @memset(m, undefined);
+            var computed_tag = state.mac(tag_bits, ad.len, m.len);
+            const verify = crypto.utils.timingSafeEql([tag_length]u8, computed_tag, tag);
+            crypto.utils.secureZero(u8, &computed_tag);
```

Comment:
Maybe the call to `secureZero()` could be moved to the `(!verify)` block, to avoid that call in the happy path.